.github/workflows/ci-build.yml

```diff
@@ -24,9 +24,28 @@ jobs:
             ${{ runner.os }}-maven-
       - name: Enable Testconatiner reuse
         run: echo 'testcontainers.reuse.enable=true' >> "$HOME"/.testcontainers.properties
+      - name: Create AWS test setup
+        run: |
+          echo "owner = \"$OWNER\"" > cloudSetup/terraform.tfvars
+          echo "deputy = \"$DEPUTY\"" >> cloudSetup/terraform.tfvars
+          terraform -chdir=cloudSetup/ init
+          terraform -chdir=cloudSetup/ apply -auto-approve
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          OWNER: ${{ secrets.AWS_TAG_OWNER }}
+          DEPUTY: ${{ secrets.AWS_TAG_DEPUTY }}
       - name: Run tests and build with Maven
-        run: mvn -B clean verify sonar:sonar --file pom.xml -Dsonar.organization=exasol -Dsonar.host.url=https://sonarcloud.io -Dsonar.login=$SONAR_TOKEN
+        run: |
+          . cloudSetup/generated/setEnv.sh
+          mvn -B verify sonar:sonar --file pom.xml -Dsonar.organization=exasol -Dsonar.host.url=https://sonarcloud.io -Dsonar.login=$SONAR_TOKEN
         env:
           GITHUB_OAUTH: ${{ secrets.GITHUB_TOKEN }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
\ No newline at end of file
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+      - name: Destroy cluster
+        run: terraform -chdir=cloudSetup/ destroy -auto-approve
+        if: always()
```

Comment:
Is this necessary?

Reply:
yes, otherwise it won't get executed if the previous one filed. And we wan't to destroy the cluster in any case, otherwise it get's costy ;-) .

Reply:
I see, maybe a comment would be good in this case.